Documentation/network/lb-ipam.rst

```diff
@@ -0,0 +1,382 @@
+          color: red
+
+There are a few special purpose selector fields which don't match on labels but
+instead on other metadata like ``.meta.name`` or ``.meta.namespace``.
+
+=============================== ===================
+Selector                        Field
+------------------------------- -------------------
+io.kubernetes.service.namespace ``.meta.namespace``
+io.kubernetes.service.name      ``.meta.name``
+=============================== ===================
+
+.. code-block:: yaml
+
+    apiVersion: "cilium.io/v2alpha1"
+    kind: CiliumLoadBalancerIPPool
+    metadata:
+      name: "blue-pool"
+    spec:
+      cidrs:
+      - cidr: "20.0.10.0/24"
+      serviceSelector:
+        matchLabels:
+          "io.kubernetes.service.namespace": "tenant-a"
+
+Conflicts
+---------
+
+IP Pools are not allowed to have overlapping CIDRs. When an administrator does
+create pools which overlap, a soft error is caused. The last added pool will be
+marked as ``Conflicting`` and no further allocation will happen from that pool.
+Therefor administrators should always check the status of all pools after making
+modifications.
+
+If we for example add 2 pools(``blue-pool`` and ``red-pool``) both with the same
+CIDR, we will see the following:
+
+.. code-block:: shell-session
+
+    $ kubectl get ippools
+    NAME        DISABLED   CONFLICTING   IPS AVAILABLE   AGE
+    blue-pool   false      false         254             25m
+    red-pool    false      true          254             11s
+
+The reason for the conflict is stated in the status and can be accessed like so
+
+.. code-block:: shell-session
+
+    $ kubectl get ippools/red-pool -o jsonpath='{.status.conflictReason}'
+    Pool conflicts since CIDR '20.0.10.0/24' overlaps CIDR '20.0.10.0/24' from IP Pool 'blue-pool'
+
+or
+
+.. code-block:: shell-session
+
+    $ kubectl describe ippools/red-pool
+    Name:         red-pool
+    #[...]
+    Status:
+      #[...]
+      Conflict Reason:  Pool conflicts since CIDR '20.0.10.0/24' overlaps CIDR '20.0.10.0/24' from IP Pool 'blue-pool'
+      Conflicting:      true
+      #[...]
+
+Disabling a Pool
+-----------------
+
+IP Pools can be disabled. Disabling a pool will stop LB IPAM from allocating
+new IPs from the pool, but doesn't remove existing allocations. This allows
+an administrator to slowly drain pool or reserve a pool for future use.
+
+.. code-block:: yaml
+
+    apiVersion: "cilium.io/v2alpha1"
+    kind: CiliumLoadBalancerIPPool
+    metadata:
+      name: "blue-pool"
+    spec:
+      cidrs:
+      - cidr: "20.0.10.0/24"
+      disabled: true
+
+.. code-block:: shell-session
+
+    $ kubectl get ippools          
+    NAME        DISABLED   CONFLICTING   IPS AVAILABLE   AGE
+    blue-pool   true       false         254             41m
+
+Status
+------
+
+The IP Pool's status contains additional counts which can be used to monitor
+the amount of used and available IPs. A machine parsable output can be gotten like so.
```

Comment:
I'm not familiar with “gotten”, but that's maybe OK. I'm nitpicking here.

```suggestion
the amount of used and available IPs. A machine parsable output can be obtained like so.
```

Reply:
`gotten` is more American English(I tend to use both since it is not my primary language), `obtained` does sound better, will go with that.